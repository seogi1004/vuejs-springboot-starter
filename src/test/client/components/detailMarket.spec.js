import { shallow } from 'vue-test-utils'
import DetailMarketComp from '@/components/detailMarket'
import { data } from '../samples.js'

describe('detailMarket.vue', () => {
    let cmp ;

    beforeEach(() => {
        cmp = shallow(DetailMarketComp, {
            propsData: {
                title: 'Min Market Cap',
                data: data[2]
            }
        });
    });

    it('snapshot', () => {
        cmp.vm.$nextTick(() => {
            expect(cmp.vm.$el).toMatchSnapshot();
        });
    });

    it('computed', () => {
        expect(cmp.vm.maxYear).toEqual('2017');
        expect(cmp.vm.minYear).toEqual('2015');
    });
});