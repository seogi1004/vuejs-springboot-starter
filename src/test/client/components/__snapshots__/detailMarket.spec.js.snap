// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`detailMarket.vue snapshot 1`] = `
<div>
  <h5>
    Min Market Cap
  </h5>
   
  <table>
    <thead>
      <tr>
        <th>
          Company
        </th>
         
        <th>
          Max Year
        </th>
         
        <th>
          Min Year
        </th>
      </tr>
    </thead>
     
    <tbody>
      <tr>
        <td>
          Facebook
        </td>
         
        <td>
          2017
        </td>
         
        <td>
          2015
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
